pages/concepts/presence.mdx

```diff
@@ -20,45 +20,51 @@ r.subscribeToPresence((presentClientIDs) => {
   `presentClientIDs` will always contain the client's own client ID.
 </Callout>
 
-## Per-Client Presence State
+## Presence State
 
-The `subscribeToPresence` method only tells you the IDs of the clients that are present.
-To do something useful with them, you will combine these IDs with additional synced data about clients.
+The `subscribeToPresence` method only tells you the client IDs of the clients that are present.
+To do something useful with them, you will combine these client IDs with additional synced data about clients.
 This additional client data is custom to your application. It is read and written
 like any other Reflect data and can be [modeled using the Rails helper library](./model).
 
 For example you might model your client data like so:
 
 ```ts
-import { generate } from "@rocicorp/rails";
+import { generatePresence } from "./generate-presence.js";
 
 type Client = {
+  clientID: string;
   id: string;
   name: string;
   cursor: { x: number; y: number };
 };
 
 export const {
-  put: putClient,
+  set: setClient,
   get: getClient,
   init: initClient,
   update: updateClient,
   list: listClients,
-} = generate<Client>("client");
+  listEntries: listClientEntries,
+} = generatePresence<Client>("client");
```

Comment:
If I'm understanding the API correctly, then the React example below needs to be updated, `await getClient(tx, clientID)` should be `await getClient(tx, {clientID});`